.github/workflows/publish.yml

```diff
@@ -57,6 +55,17 @@ jobs:
         env:
           BAIPP_DIST: ${{ steps.baipp.outputs.dist }}
 
+      # Run tests on the built package (which will be later uploaded to PyPI)
+      - name: "Install PST from wheel and test"
+        env:
+          BAIPP_DIST: ${{ steps.baipp.outputs.dist }}
+        run: |
+          dist_loc="${BAIPP_DIST}"
+          # calling with --installpkg so we can use the already built package
+          tox run -e py312-tests-no-cov \
+          --installpkg $dist_loc/*.whl \
+          -- --deselect tests/test_build.py::test_translations
+
```

Comment:
I added this to the prerelease workflow but should have added them to the publish one too for some final quality checks on the packaged artefacts